salt/modules/cmdmod.py

```diff
@@ -873,6 +874,9 @@ def run(cmd,
     :param bool encoded_cmd: Specify if the supplied command is encoded.
       Only applies to shell 'powershell'.
 
+    :pram bool raise_err: Specifies whether to raise a CommandExecutionError.
+      If False, the error will be logged. Default is False.
```

Comment:
This is a little unclear. Should it read `If False, the error will be logged and no exception will be raised`?